EventFilter/CastorRawToDigi/src/ZdcUnpacker.cc

```diff
@@ -12,236 +13,278 @@
 #include "FWCore/MessageLogger/interface/MessageLogger.h"
 #include <map>
 
-namespace ZdcUnpacker_impl {
+
+namespace HcalUnpacker_impl {
```

Comment:
Is this a plain copy of HcalUnpacker_impl from HcalRawToDigi/src/HcalUnpacker.cc?
If so, we should use that code instead.
If it's different, please rename back to ZdcUnpacker_impl